frontend/test/metabase/scenarios/collections/collections-sidebar.cy.spec.js

```diff
@@ -0,0 +1,47 @@
+import { restore } from "__support__/e2e/cypress";
+
+describe("collections sidebar (metabase#15006)", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.visit("/collection/root");
+  });
+
+  describe("in desktop form factors", () => {
+    it("should display collections sidebar on page load", () => {
+      cy.findByText("First collection").should("be.visible");
```

Comment:
Countless other tests in collections are already checking this so this particular test will not give us any more confidence.

`cy.findByText()` is an implicit assertion in itself. No need to add `.should("be.visible")` to it.